### DISCUSSION TITLE
foliumap.add_local_tiler() -> foliumap.to_streamlit() doesn't show up the raster tiled

### DISCUSSION THREAD

**Discussion by PavlosDem99** [opengeos/leafmap#339] [Q&A] [answered]
Hello 
I am facing a problem with foliumap.add_local_tiler(). The tiled raster doesn't show up on the folium Map. 

![image](https://user-images.githubusercontent.com/95135614/211145799-bebd72fc-c2ac-4362-a895-5f492101e541.png)

**Comment by giswqs** [comment, marked as answer]
Localtileserver doesn't support streamlit. Don't waste your time on this
